NASA ADS.js

```diff
@@ -1,15 +1,15 @@
 {
 	"translatorID": "7987b420-e8cb-4bea-8ef7-61c2377cd686",
 	"label": "NASA ADS",
-	"creator": "Tim Hostetler, Abe Jellinek, and Zoë C. Ma",
-	"target": "^https://ui\\.adsabs\\.harvard\\.edu/(search|abs)/",
+	"creator": "Tim Hostetler, Abe Jellinek, Zoë C. Ma, Jennifer Chen",
+	"target": "^(https://ui\\.adsabs\\.harvard\\.edu/(search|abs)/)|(^https://scixplorer\\.org/(search|abs/))",
```

Comment:
Can simplify if these have the same path structure:

```suggestion
	"target": "^https://(ui\\.adsabs\\.harvard\\.edu|sciexplorer\\.org)/(search|abs)/",
```

Reply:
Not quite the same, for search in ADS:
```https://ui.adsabs.harvard.edu/search/...```
for SciX:
```https://scixplorer.org/search?.....```